Orange/widgets/data/owfile.py

```diff
@@ -176,7 +176,7 @@ def __init__(self):
 
         box = gui.hBox(self.controlArea)
         gui.button(
-            box, self, "Browse documentation data sets",
+            box, self, "Browse Documentation Data Sets",
```

Comment:
What about breaking the rule here? This button is so long that title case looks weird.